sdk/cosmos/azure-cosmos-spark_3-1_2-12/README.md

```diff
@@ -131,6 +134,7 @@ https://github.com/Azure/azure-sdk-for-java/issues/new
 #### azure-cosmos-spark_3-4_2-12
 | Connector | Supported Spark Versions | Minimum Java Version | Supported Scala Versions | Supported Databricks Runtimes |
 |-----------|--------------------------|----------------------|--------------------------|-------------------------------|
+| 4.24.0    | 3.4.0                    | 8                    | 2.12                     | 11.\*, 12.\*                  |
```

Comment:
This is wrong - 13.* is Spark 3.4 - probably worth fixing in this PR? Same for other SPark versions

Reply:
Done, thanks!